crates/common/fork_choice/src/handlers.rs

```diff
@@ -198,7 +207,7 @@ pub async fn on_block(
         .insert(block.tree_hash_root(), is_timely);
 
     // Add proposer score boost if the block is timely and not conflicting with an existing block
-    let is_first_block = store.proposer_boost_root == block_root;
+    let is_first_block = store.proposer_boost_root == B256::default();
```

Comment:
```suggestion
    let is_first_block = store.proposer_boost_root == B256::ZERO;
```

default defaults to 0 but this is a bit more readable